index.html

```diff
@@ -1,196 +1,228 @@
+                <li>CANOPY</li>
+                <li class="common">Back End Dev</li>
+                <li class="common">2015</li>
+              </ul>
+
+              <p class="paragraph">
+                A daily selection of privately personalized reads no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>Html</li>
+                <li>Css</li>
+                <li>Javascript</li>
+              </ul>
+              <a class="see-project" data-index-number="2" href=""
+                >See Project</a
+              >
+            </div>
           </li>
-          <li>
-            Css
+
+          <li class="card-details">
+            <div>
+              <img src="./photos/Portfolio2.png" alt="tonic" />
+            </div>
+            <div class="text-div">
+              <h2 class="title">Multi-Post Stories</h2>
+              <ul class="tech-btns">
+                <li>CANOPY</li>
+                <li class="common">Back End Dev</li>
+                <li class="common">2015</li>
+              </ul>
+
+              <p class="paragraph">
+                A daily selection of privately personalized reads no accounts or
+                sign-ups required.
+              </p>
+              <ul class="languages">
+                <li>Html</li>
+                <li>Css</li>
+                <li>Javascript</li>
+              </ul>
+              <a class="see-project" data-index-number="2" href=""
+                >See Project</a
+              >
+            </div>
           </li>
-          <li>Javascript</li>
         </ul>
-          <a class="see-project" data-index-number="2" href="">See Project</a>
-      
-    </div>
-  </li>
-</ul>
-    </section>
-  </main>
-</body>
-
-</html>
\ No newline at end of file
+      </section>
+      <section class="contact-form">
+        <div class="container">
+          <div class="description-contact">
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+          <form
+            action="https://formspree.io/f/mvolakkg"
+            method="POST"
+            class="form"
+          >
+            <input
+              type="text"
+              name="name"
+              placeholder="Name"
+              maxlength="30"
+              class="form"
+              id="name"
+              required
+            />
+            <input
+              type="email"
+              name="email"
+              placeholder="Email"
+              class="form"
+              id="email"
+              required
+            />
+            <textarea
+              name="message"
+              id="textarea"
+              placeholder="Write your message here"
+              maxlength="500"
```

Comment:
- [ ] Let's make sure we have `required` on the comments textarea message as we don't want to get emails with an empty message.